server/conf/timely-standalone.yml

```diff
@@ -0,0 +1,47 @@
+timely:
+  ip: 0.0.0.0
+  port:
+    put: 54321
+    query: 54322
+    websocket: 54323
+  instance-name: TimelyStandalone
```

Comment:
I think re-org should be a different ticket. Here's why - I think that a PR for a feature/improvement should be judged based on what it's trying to do. Having someone change past mistakes, unrelated to their change, in their PR is scope creep and is very annoying (I know this from past experience).